apollo-router/src/plugins/telemetry/apollo_exporter.rs

```diff
@@ -83,14 +88,14 @@ impl Default for Sender {
 /// The Apollo exporter is responsible for attaching report header information for individual requests
 /// Retrying when sending fails.
 /// Sending periodically (in the case of metrics).
-#[derive(Clone)]
 pub(crate) struct ApolloExporter {
     batch_config: BatchProcessorConfig,
     endpoint: Url,
     apollo_key: String,
     header: proto::reports::ReportHeader,
     client: Client,
-    strip_traces: Arc<Mutex<bool>>,
+    strip_traces: Mutex<bool>,
```

Comment:
Why not just an `AtomicBool` ?

Reply:
done: b6c8d80870eee01e496d03db4d93888662811baf